src/main/java/com/github/zly2006/reden/utils/Utils.kt

```diff
@@ -30,6 +37,11 @@ fun PlayerEntity.sendMessage(s: String) {
     sendMessage(Text.literal(s))
 }
 
+val ClientPlayerEntity.handToolItem: Boolean get() {
+    val stack = getStackInHand(Hand.MAIN_HAND) ?: return false
+    return Registries.ITEM.getId(stack.item) == Identifier.tryParse(SELECTION_TOOL.stringValue)
+}
```

Comment:
`handToolItem` sounds like a `Item` rather than a `Boolean`. Suggest changing to something like `holdingToolItem`.